projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/interact/firestarter.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:firestarter",
+  "text": [
+    {
+      "text": "只要纵火能力发动的话，一切都会好转的。"
```

```suggestion
      "text": "纵火能力的发动，使一切发生了天翻地覆的变化。"
```
这里稍微有些问题。原句出自《降世神通》系列，但是我没看过，所以按照原本语境改了改语序。
原句：Long ago, the four nations lived together in harmony. Then everything changed when the Fire Nation attacked.

---

淦看过降世神通没看出来
这里其实是《帝国的毁灭》里的台词